### PR TITLE
[12.0][IMP] hr_timesheet_sheet: Clearer interface

### PR DESCRIPTION
**Pull request by tarteo**
Same as #239 but for 12.0

### CODE REVIEW

File: hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
                                     <tree limit="1000000">
                                         <field name="value_x"/>
                                         <field name="value_y"/>
-                                        <field name="unit_amount"/>
+                                        <field name="unit_amount" widget="float_time"/>
```
Comment (alexey-pelykh) [inline hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml:67]: Should this be somehow configurable?

**Review by alexey-pelykh — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Isn't this better done on the widget?

**Comment by alexey-pelykh** [comment]
I think it's very usecase-dependant

**Comment by pedrobaeza** [comment]
I mean the style.

**Comment by alexey-pelykh** [comment]
Same as #239 which is merged already. I think that it should be widget-dependant, yet no principal position there

**Comment by alexey-pelykh** [comment]
@pedrobaeza so I think for consistency we should just merge and figure out in next improvement?

**Comment by tarteo** [comment]
I think for every use case the colors etc will be different it will be very hard to cover all the use cases in a generic option.

**Comment by pedrobaeza** [comment]
OK, it was only a reflection to see if it's worth.

**Pull request merged**